extensions/udapdr/scripts/DSP_Multiple_Reranker.py

```diff
@@ -0,0 +1,690 @@
+
+		evaluate_dataset(chosen_LoTTE_type, chosen_LoTTE_split, chosen_LoTTE_set, 5, 
+			             "../ColBERT_FM/downloads/lotte/" + chosen_LoTTE_split + "/" + chosen_LoTTE_set + "/qas." + chosen_LoTTE_type + ".jsonl", 
+			             distilled_ranking)
+
+		evaluate_dataset(chosen_LoTTE_type, chosen_LoTTE_split, chosen_LoTTE_set, 20, 
+			             "../ColBERT_FM/downloads/lotte/" + chosen_LoTTE_split + "/" + chosen_LoTTE_set + "/qas." + chosen_LoTTE_type + ".jsonl", 
+			             distilled_ranking)
+
+		evaluate_dataset(chosen_LoTTE_type, chosen_LoTTE_split, chosen_LoTTE_set, 100, 
+			             "../ColBERT_FM/downloads/lotte/" + chosen_LoTTE_split + "/" + chosen_LoTTE_set + "/qas." + chosen_LoTTE_type + ".jsonl", 
+			             distilled_ranking)
+	
+	elif LoTTE_or_BEIR == "BEIR":
+
+		evaluate_dataset(chosen_LoTTE_type, chosen_LoTTE_split, chosen_LoTTE_set, 5, 
+			             "../ColBERT_FM/beir_datasets/" + chosen_BEIR_set + "/" + chosen_BEIR_type + "/qas.jsonl", 
+			             distilled_ranking)
+
+		evaluate_dataset(chosen_LoTTE_type, chosen_LoTTE_split, chosen_LoTTE_set, 20, 
+			             "../ColBERT_FM/beir_datasets/" + chosen_BEIR_set + "/" + chosen_BEIR_type + "/qas.jsonl", 
+			             distilled_ranking)
+
+		evaluate_dataset(chosen_LoTTE_type, chosen_LoTTE_split, chosen_LoTTE_set, 100, 
+			             "../ColBERT_FM/beir_datasets/" + chosen_BEIR_set + "/" + chosen_BEIR_type + "/qas.jsonl", 
+			             distilled_ranking)
+
+		evaluate_beir(distilled_ranking, chosen_BEIR_set, chosen_BEIR_type, downloads_folder)
+
+######################################################################
+
+
+
+
+
+if __name__ == '__main__':
+
+	parser = argparse.ArgumentParser()
+
+	parser.add_argument("--chosen_LoTTE_split", type=str, required=True)
+	parser.add_argument("--chosen_LoTTE_type", type=str, required=True)
+	parser.add_argument("--chosen_LoTTE_set", type=str, required=True)
+
+	parser.add_argument("--LoTTE_or_BEIR", type=str, required=True)
+	parser.add_argument("--chosen_BEIR_set", type=str, default=None, required=True)
+	parser.add_argument("--chosen_BEIR_type", type=str, default=None, required=True)
+
+	parser.add_argument("--sample_count", type=int, required=True)
+	parser.add_argument("--reranker_count", type=int, default=5, required=True)
+	parser.add_argument("--devices", type=list, default=["cuda:0", "cuda:1", "cuda:2", "cuda:3", "cuda:4"], required=False)
+
+	parser.add_argument("--query_count", type=int, required=True)
+	parser.add_argument("--model_choice", type=str, required=True)
+	parser.add_argument("--gpt3_model_choice", type=str, required=True)
+	parser.add_argument("--parallelization", type=bool, required=True)
+	parser.add_argument("--dsp_prompting", type=bool, required=True)
+	parser.add_argument("--use_FLAN_for_all_synthetic_query_generation", default=False, type=bool, required=False)
+	parser.add_argument("--downloads_folder", type=str, default="../downloads", required=True)
+
```

Comment:
Please add some descriptions of the arguments, how its related to the downloaded data.